cprover_bindings/src/goto_program/typ.rs

```diff
@@ -46,6 +46,8 @@ pub enum Type {
     IncompleteStruct { tag: InternedString },
     /// `union x {}`
     IncompleteUnion { tag: InternedString },
+    /// `integer`
```

Comment:
What is the difference between this and CInteger?

Reply:
This one's a machine independent integer. Added that in the comment.